src/user_preferences.jl

```diff
@@ -0,0 +1,4 @@
+global MIN_CHUNK_SIZE = 40
+global MAX_CHUNK_SIZE = 256
+global MODEL = "text-embedding-3-large"
+global EMBEDDING_SIZE = 1024
```

Comment:
I thought your research was that we should always be the full size 3072. It can always be truncated to 1024.

Also, if you create knowledge packs and set `-0-` in the filename, that implies it's not truncated! So it has to be embedded with 3072 size (no truncation). 

We can then create the 1024 version later.

Reply:
On the chunk size, shall we use 384 default? It would be good to add a one-liner what they do -- eg, max_chunk_size is the target chunk size for the splitter. the min_chunk_size is to remove useless/small bits.

Reply:
Yes, the results look good. setting the default to 3072 and 384